application/send_email/tests/test_send_email.py

```diff
@@ -5,11 +5,11 @@
 
 from aws_lambda_powertools.utilities.typing import LambdaContext
 from pytest import fixture, raises
+from send_email.send_email import lambda_handler, send_email
 
-from application.common.types import EmailMessage
-from application.send_email.send_email import lambda_handler, send_email
+from common.types import EmailMessage
 
-FILE_PATH = "application.send_email.send_email"
+FILE_PATH = "send_email.send_email"
```

Comment:
Is there a reason this has changed? I believe it stops sonarcloud from properly tracking coverage